pkg/store/tsdb.go

```diff
@@ -165,10 +165,20 @@ func (s *TSDBStore) translateAndExtendLabels(m, extend labels.Labels) []storepb.
 }
 
 // LabelNames returns all known label names.
-func (s *TSDBStore) LabelNames(ctx context.Context, r *storepb.LabelNamesRequest) (
+func (s *TSDBStore) LabelNames(ctx context.Context, _ *storepb.LabelNamesRequest) (
```

Comment:
Missing handling of partial response here.

Reply:
Same as above of comment.

Reply:
Yeah sadly tsdb doesn't provide context.Context support, so feel free to ignore this comment :+1: